v2/en/changelog-1015006.md

```diff
@@ -0,0 +1,118 @@
+    @creator       Thibault Charbonnier
+    @created       2018-11-14 01:32 GMT
+--->
+
+# Version 1.15.6.2 - dd MMMM YYYY
+
+* upgraded the [nginx](nginx.html) core to 1.15.6.
+    * see the changes here: http://nginx.org/en/CHANGES
+* feature: ./configure: added new options --without-stream_ssl_module and --without-stream.
+* feature: ./configure: support @rpath placeholder in OS X. Thanks Datong Sun for the patch.
+* bugfix: [nginx](nginx.html) did not destroy the cycle memory pool before the daemon process exits. Thanks Yuansheng Wang for the patch.
+* resty-doc indexes: skipped nginx's njs docs due to the use of special xml tags.
+* upgraded [ngx_lua](https://github.com/openresty/lua-nginx-module#readme) to 0.10.14rc3.
+    * change: we no longer support the standard Lua 5.1 interpreter (PUC-Rio Lua). It is now strongly recommended to use the OpenResty LuaJIT releases, bundled with the official OpenResty releases.
+    * change: we now print an alert when a non openresty-specific version of LuaJIT is detected since many optimizations would be missing. Thanks Thibault Charbonnier for the patch.
+    * change: we now print a warning when writing to the Lua global variable _G.
+    * feature: added support for ARM64 (Aarch64). Thanks Cloudflare for sponsoring this work. Thanks Dejiang Zhu and Zexuan Luo for the development work of this patch.
+    * feature: implemented the receiveany() upstream cosocket method. Thanks spacewander for the patch.
+    * feature: implemented a new 'lua_sa_restart' directive, which sets the SA_RESTART flag on [nginx](nginx.html) workers signal dispositions. Thanks Thibault Charbonnier for the patch.
+    * bugfix: fixed segfault in [NGINX](nginx.html) core >= 1.15.0 when [init_worker_by_lua*](https://github.com/openresty/lua-nginx-module#init_worker_by_lua) is used. Thanks Datong Sun for the patch.
+    * bugfix: [ngx.process](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/process.md#readme): [type()](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/process.md#type) didn't return 'master' in master process. Thanks spacewander for the patch.
+    * bugfix: [tcpsock:setkeepalive](https://github.com/openresty/lua-nginx-module#tcpsocksetkeepalive): worker processes might take too long to gracefully shut down when the keep alive connections take a long max idle time. Thanks Dejiang Zhu for the patch.
+    * bugfix: when variable is cacheable and indexed, setting the request "Host" header should clear any existing cached $host variable value so that subsequent reads return the expected result.
+    * bugfix: silenced -Wcast-function-type warnings, allowing for compilation with gcc8. Thanks Alessandro Ghedini for the patch.
+    * bugfix: the 'dd' calls in ngx_http_lua_util.h are now guarded by '#ifdef DDEBUG'. Thanks spacewander for the patch.
+    * optimize: removed a duplicated include directive in ngx_http_lua_regex.c. Thanks Thibault Charbonnier for the patch.
```

Comment:
This is too trivial to mention. Better remove other similarly trivial entries (if any).